### DISCUSSION TITLE
Request: Xbox GamePass Store Login

### DISCUSSION THREAD

**Discussion by Rumik** [Heroic-Games-Launcher/HeroicGamesLauncher#2782] [Ideas]
Hello! Big fan of the newly added ability to add Browser games to your library. I have been doing this manually on Steam Deck for months, so this does speed that up somewhat. However, you still need to get the link to the game from the browser in order to paste it into Heroic Launcher.

Would it be possible to add the Xbox GamePass library via Heroic's Stores menu to make it easy to browse and add play games from there?

**Comment by imLinguin** [comment]
With addition of Browser sideloading you can now add web pages to Heroic. 

You can take a look at this showcase of the feature by GamingOnLinux:
https://www.youtube.com/watch?v=j8YdKoC6-nE

I think pre-added browser apps are out of scope of the project at this moment
